libs/cde-visualization/src/index.ts

```diff
@@ -1 +1,17 @@
+import { createApplication } from '@angular/platform-browser';
+import { createCustomElement } from '@angular/elements';
+import { CdeVisualizationComponent } from './lib/cde-visualization/cde-visualization.component';
+
+(async () => {
```

Comment:
Annoyingly @angular/elements have not been updated to work with signal based components yet... I'm currently working on a utility library that will make this simpler and work around the bug. Meanwhile continue to use the component as a regular angular component instead of trying to use it as a web component.